integration-tests/account/remove_zero_balance.move

```diff
@@ -0,0 +1,24 @@
+//# init -n dev
+
+//# faucet --addr alice --amount 0
```

Comment:
amount 0 will cause no gas fee for executing the transaction, and the transaction will be discard